include/operations/blas_operators.hpp

```diff
@@ -147,9 +147,7 @@ SYCLBLAS_DEFINE_UNARY_OPERATOR(iniPrdOp1_struct,
                                (constant<R, const_val::one>::value))
 SYCLBLAS_DEFINE_UNARY_OPERATOR(posOp1_struct, (r))
 SYCLBLAS_DEFINE_UNARY_OPERATOR(negOp1_struct, (-r))
-SYCLBLAS_DEFINE_UNARY_OPERATOR(
-    sqtOp1_struct,
-    (static_cast<double>(cl::sycl::sqrt(static_cast<double>(r)))))
+SYCLBLAS_DEFINE_UNARY_OPERATOR(sqtOp1_struct, (cl::sycl::sqrt(r)))
```

Comment:
I've reviewed a separate MR that has this patch? is this re-appearing?

Reply:
It has been merged in master but I don't know why it is again shown here. I checked the master file and it shows that it is there.